vsintegration/Vsix/VisualFSharpOpenSource/Source.extension.vsixmanifest

```diff
@@ -2,7 +2,7 @@
 <!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="VisualFSharp" Version="15.4.1.9055" Language="en-US" Publisher="Microsoft.VisualFSharpTools" />
+    <Identity Id="VisualFSharp" Version="42.42.42.42" Language="en-US" Publisher="Microsoft.VisualFSharpTools" />
```

Comment:
Yes, this was intentional.  For debugging to work with this VSIX package, the version has to be bigger than whatever will be in VisualFSharpFull.vsix, and having come from the Roslyn team I opted to do what they did and re-use 42.42.42.42.